features/iotserver-feature/org.wso2.carbon.device.mgt.iot.feature/src/main/resources/jaggeryapps/store/extensions/app/store-device-type/themes/store/helpers/dashboard.js

```diff
@@ -0,0 +1,24 @@
+/*
+ *  Copyright (c) 2005-2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Fixed in [#53f17f4](https://github.com/wso2-incubator/iot-server-extensions/commit/53f17f44e2f0c8be30f944958530e12dac3843b2). JS dropped.